app/main.py

```diff
@@ -1 +1,13 @@
-# write your code here
+def copy_file(command: str) -> None:
+    copy, file_name, new_file_name = command.split()
+    if copy != "cp":
+        raise AttributeError(f"Name of copy command "
```

Comment:
are u sure about this [exception](https://www.geeksforgeeks.org/python-attributeerror/#:~:text=AttributeError%20can%20be%20defined%20as%20an%20error%20that%20is%20raised%20when%20an%20attribute%20reference%20or%20assignment%20fails.%C2%A0) ?

Reply:
AttributeError can be defined as an error that is raised when an attribute reference or assignment fails. 
For example, if we take a variable x we are assigned a value of 10. In this process suppose we want to append another value to that variable. It’s not possible. Because the variable is an integer type it does not support the append method

Seems more like ValueError)), but  I didn't define it with exact definition, 'cause, if it's command which is assigned to some method inside class, it seems to me more logical, that we cant attach some other command 'cause it shouldn't support this method. Just wanted to get some answers. Pretty sure it's ValueError)